### PR TITLE
Restored buttons in UCR to view data source and report config sources

### PR DESCRIPTION
**Pull request by orangejenny**
Restores "View Source" buttons for data sources and report configs, which were just removed in https://github.com/dimagi/commcare-hq/pull/25641/ Basically reverts 77dacae1724c146fd7bbfb74cbd13244376d17f3 and 5f843e449bd52cee82d2240a6e4b5522125b9b4a but adds analytics and accommodates UI changes.

https://github.com/dimagi/commcare-hq/pull/25641#issuecomment-543128153

Feature flag: UCR

### CODE REVIEW

**Review by mkangia — APPROVED** [review]

File: corehq/apps/userreports/templates/userreports/edit_data_source.html [hunk 1]
```diff
@@ -68,6 +68,11 @@ <h5 class="modal-title">{% trans 'Are you sure you want to delete this data sour
       {% endif %}
       <a href="{% url 'preview_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Preview data' %}</a>
       <a href="{% url 'summary_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Data Source Summary' %}</a>
+      <a href="{% url 'configurable_data_source_json' domain data_source.get_id %}"
+         class="btn btn-default track-usage-link"
```
Comment (mkangia) [inline corehq/apps/userreports/templates/userreports/edit_data_source.html]: just confirming that `track-usage-link` class and other data attrs make the tracking happen?

**Review by mkangia — APPROVED** [review]

File: corehq/apps/userreports/templates/userreports/edit_data_source.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/userreports/templates/userreports/edit_data_source.html]: yep: https://github.com/dimagi/commcare-hq/blob/99187d23fbf7669c3e816a223fa92273bab0c830/corehq/apps/hqwebapp/static/hqwebapp/js/main.js#L350-L354

Comment (orangejenny) [reply corehq/apps/userreports/templates/userreports/edit_data_source.html]: yep: https://github.com/dimagi/commcare-hq/blob/99187d23fbf7669c3e816a223fa92273bab0c830/corehq/apps/hqwebapp/static/hqwebapp/js/main.js#L350-L354

**Comment by czue** [comment]
:+1:

**Pull request merged**